guides/v2.3/config-guide/mq/manage-message-queues.md

```diff
@@ -57,7 +57,7 @@ Edit */app/etc/env.php* file for configure cron job `consumers_runner`
 ...
 ```
 
-* `cron_run` - the option for enabling/disabling cron job `consumers_runner`, by default is true.
+* `cron_run` - the option for enabling/disabling cron job `consumers_runner`, by default is false.
 * `max_messages` - the maximum number of messages for each consumer that must be processed before consumer terminate, by default is 1000. If it is 0, then the consumer never stops working.
```

Comment:
```suggestion
* `max_messages` - A number specifying the maximum number of messages each consumer must process before terminating (default = `1000`). Although we do not recommend it, you can use 0 to prevent the consumer from terminating.
```